src/tests/regressionTests.test.tsx

```diff
@@ -227,11 +228,11 @@ const checkpoint = (name: string) => {
   );
 };
 
-beforeEach(() => {
+beforeEach(async () => {
   // Unmount ReactDOM from root
   ReactDOM.unmountComponentAtNode(document.getElementById("root")!);
 
-  localStorage.clear();
+  await h.storage.clear();
```

Comment:
@dwelle We seem to be doing `window.storage.clear()` here. I'm just curious, the instance of `StorageProvider` is what we are referencing here right?

Reply:
Yes. `h.storage` is a reference to `storage` imported in `App.tsx` for this purpose.

Reply:
@dwelle So I think I have a hunch on why IDB is not clearing. This probably goes back to `jsdom` as I had mentioned earlier in the PR. We seem to be checking `indexedDB in window` where window is coming from JSDOM, which doesnt support IDB.

Here's a quick codesandbox that shows JSDOM doesnt have this property. https://codesandbox.io/s/jsdom-testing-jnq9r

Reply:
No, I tried it. And it's set correctly to true even implicitly because the polyfill  I added takes care of it.